samples/Nancy.Swagger.Annotations.Demo/Bootstrapper.cs

```diff
@@ -18,6 +18,14 @@ protected override void ApplicationStartup(TinyIoCContainer container, IPipeline
                 EmailAddress = "exampleEmail@example.com"
             });
 
+            SwaggerMetadataProvider.SetSwaggerRoot(
+                externalDocumentation: new ExternalDocumentation {
+                    Description = "GitHub",
+                    Url = "https://github.com/yahehe/Nancy.Swagger"
+                }, 
+                schemes: new[] { Schemes.Http } 
```

Comment:
I assume HTTP encompasses HTTPS here?

Reply:
If it does this is good to merge, if not can we put HTTPS here instead?

Reply:
No. Http is not the same as Https in the Schemes enum. 
But when someone runs this sample, it's not going to be running on HTTPS - it defaults to HTTP.
So if they want to test the endpoints, HTTP is needed.

Reply:
I guess that's a fair point